packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -247,7 +207,19 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable {
         return _stampAssertion(assertionId);
     }
 
-    function getMinimumBond(address currencyAddress) public view returns (uint256) {
+    function getMinimumBond(address currencyAddress) external view nonReentrantView() returns (uint256) {
```

Comment:
This would prevent someone from calling this function from a `_callbackOnAssertionDispute` or `_callbackOnAssertionResolve` right? Won't this `nonReentrantView` make things harder for the `callbackRecipient` to make a new `assertTruthFor` on the callback?

What are the reasons for having a `nonReentrantView` here?